com.mudita.mudita-center.yaml

```diff
@@ -0,0 +1,37 @@
+app-id: com.mudita.mudita-center
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+command: run.sh
+separate-locales: false
+rename-desktop-file: Mudita Center.desktop
+rename-icon: Mudita Center
+finish-args:
+  - --share=network
+  - --device=all
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=xdg-download
+modules:
+  - name: muditacenter
+    buildsystem: simple
+    sources:
+      - type: file
+        only-arches:
+          - x86_64
+        url: https://github.com/mudita/mudita-center/releases/download/2.5.0/Mudita-Center.AppImage
+        sha256: 0fbef5c2020a8f9574d438741c5907f00a828c48927b48f8b12817881a2176a5
+      - type: file
+        path: run.sh
+    build-commands:
+      - chmod +x Mudita-Center.AppImage
+      - ./Mudita-Center.AppImage --appimage-extract
+      - desktop-file-edit --set-key=Exec --set-value="run.sh %U" squashfs-root/Mudita\ Center.desktop
+      - rm -r squashfs-root/usr/share/icons/hicolor/1024x1024
+      - mv squashfs-root/* ${FLATPAK_DEST}/
+      - install -Dm644 "${FLATPAK_DEST}/Mudita Center.desktop" "${FLATPAK_DEST}/share/applications/Mudita Center.desktop"
```

Comment:
is there a metainfo file in here? I don't see it being installed.

Reply:
Created one, hope it works ok.